man/bemenu.1.in

```diff
@@ -0,0 +1,314 @@
+and
+.Ar size
+to be used.
+For more options, consult the
+.%T Pango Reference Manual
+for
+.Fn pango_font_description_from_string .
+.El
+.Pp
+The following options control the colours for various parts of the list
+for the Wayland and X11 backends.
+Each takes an argument in the form of:
+.Pp
+.Dl Cm # Ns Ar RGB Ns Op Ar A
+.Pp
+where
+.Ar R, G, B
+and
+.Ar A
+are hexadecimal integers from 00\(enFF that control the red, green, blue
+and alpha-transparency channels.
+.Bl -tag -width "--xxx color "
+.It Fl -tb Ar color
+Title background.
+.It Fl -tf Ar color
+Title foreground.
+.It Fl -fb Ar color
+Filter background.
+.It Fl -ff Ar color
+Filter foreground
+.It Fl -nb Ar color
+Normal background.
+.It Fl -nf Ar color
+Normal foreground.
+.It Fl -hb Ar color
+Highlighed background.
+.It Fl -hf Ar color
+Highlighed foreground.
+.It Fl -sb Ar color
+Selected background.
+.It Fl -sf Ar color
+Selected foreground.
+.It Fl -scb Ar color
+Scrollbar background.
+.It Fl -scf Ar color
+Scrollbar foreground.
+.El
+.Sh KEYBOARD COMMANDS
+.Nm
+supports keyboard commands to move around the list and edit the filter.
+In the following examples,
+.Cm C-x
+means
+.Aq Cm control-x ,
+.Cm M-x
+means
+.Aq Cm alt-x
+and
+.Cm S-x
+means
+.Aq Cm shift-x :
+.Bl -tag -width Ds
+.It Eo
+.Aq Cm cursor-up ,
+.Cm S- Ns Aq Cm cursor-left ,
+.Cm C-p , M-j , M-l Ec
+Highlight the previous item.
+.It Eo
+.Aq Cm cursor-down ,
+.Aq Cm tab ,
+.Cm C-n , M-h , M-k Ec
+Highlight the next item.
+.It Cm PageUp , M-v , M-u
+Show the previous page of items, preserving the offset of the currently
+highlighed item, if possible.
+.It Cm PageDown , C-v , M-d
+Show the next page of items, preserving the offset of the currently
+highlighed item, if possible.
+.It Cm S-M-< , S-PageUp
+Highlight the first item in the list.
+.It Cm S-M-> , S-PageDown
+Highlight the last item in the list.
+.It Cm S- Ns Aq Cm tab
+Copy the highlighed item to the filter.
+.It Eo
+.Aq Cm escape ,
+.Cm C-g Ec
+Terminate
+.Nm
+without printing items.
+.It Cm C- Ns Aq Cm carriage-return
+Select the highlighted item and continue highlighting.
+.It Eo
+.Cm S- Ns Aq Cm carriage-return ,
+.Cm C-m Ec
+Print selected items and the currently highlighed one, and exit.
+Items are printed in the order they were selected,
+with the currently highlighted item printed last.
+.It Eo
+.Cm S- Ns Aq Cm carriage-return ,
+.Cm Insert Ec
+Print the filter text to standard output and exit.
+.It Cm C-l , Aq Cm cursor-left
+Move cursor left.
+.It Cm C-f , Aq Cm cursor-right
+Move cursor right.
+.It Cm C-a , Home
+Move cursor to the start.
+.It Cm C-e , End
+Move cursor to the end.
+.It Eo
+.Aq Cm backspace
+.Cm C-h Ec
+Remove character before the cursor.
+.It Aq Cm delete
+Remove character at the cursor.
+.It Cm C-u , S- Ns Aq Cm delete
+Rrmove all characters before the cursor.
+.It Cm C-k
+Remove all characters after the cursor.
+.It Cm C-w
+Clear the filter.
+.It Cm M- Ns Op 0\(en9
+Print selected items and exit with a custom error code.
```

Comment:
I also couldn't trigger this on any of the backends using either alt key.

Reply:
I think that only works with wayland and x11. Also needs bemenu from git.